server/terraform/README.md

```diff
@@ -25,22 +25,54 @@ This uses the `terraform@who-terraform-admin.iam.gserviceaccount.com` service ac
 - [Terraform adding credentials](https://www.terraform.io/docs/providers/google/guides/getting_started.html#adding-credentials)
 - [Securely handling keys](https://cloud.google.com/iam/docs/understanding-service-accounts?_ga=2.87249435.-2051693357.1581897767#managing_service_account_keys)
 
-### Terraform service account
+### Terraform Service Account
 
-Only needed for terraform service account setup. The account must have these permissions:
+Service accounts are used to access the cloud projects. Development account
+needs to create projects and therefore needs these permissions:
 
 - Billing Account User
 - Project Creator
 
+#### Production Special setup
+
+Production Service Accounts only have edit access to the production project but
+needs this permission for the project (starting point is empty project):
```

Comment:
What does 'starting point is empty project' mean?

Reply:
See update comments. Point is that the project is fresh and completely empty... and that the only configuration is done through Terraform